deepspeed/runtime/zero/stage_1_and_2.py

```diff
@@ -2427,7 +2427,9 @@ def estimate_zero2_model_states_mem_needs(total_params,
         gpu_mem = 2 * total_params
         cpu_mem = total_params * max(4 * total_gpus, 16) * additional_buffer_factor
     else:
-        gpu_mem = 4 * total_params + int(16 * total_params / total_gpus)
+        # GPU's total_params multipliers: 2 = params_16bit,
+        # 14 = 2_grads_16bit + 4_params_32bit + 8_optimizer_states_32bit(momentum and variance)
+        gpu_mem = 2 * total_params + int(14 * total_params / total_gpus)
```

Comment:
```suggestion
        gpu_mem = 2 * total_params + int(18 * total_params / total_gpus)
```

Reply:
@nelyahu, apologies for the delay in resolving this. I have made suggestions to estimate 6bytes for gradients as that more closely aligns with the current implementation.